### PR TITLE
Upgrade phpstan and fix some new issues

### PR DESCRIPTION
**Pull request by Jalle19**
Replaces #305, /cc @symm

### CODE REVIEW

**Comment by Jalle19** [comment]
No idea how but I didn't get any of those league/container errors locally

File: src/Schema/DefinitionBuilder.php [hunk 1]
```diff
@@ -529,7 +529,11 @@ protected function getNamedTypeNode(TypeNodeInterface $typeNode): NamedTypeNodeI
             $namedType = $namedType->getType();
         }
 
-        /** @var NamedTypeNodeInterface $namedType */
+        if (!$namedType instanceof NamedTypeNodeInterface) {
+            throw new LanguageException(sprintf('Wrapped type must be an instance of %s',
```
Comment (crisu83) [inline src/Schema/DefinitionBuilder.php]: Dot missing from the error message. (nit-pick)

File: src/Util/ValueConverter.php [hunk 2]
```diff
@@ -70,12 +70,6 @@ public static function convert($value, TypeInterface $type): ?ValueNodeInterface
             return self::convertListType($value, $type);
         }
 
-        // Populate the fields of the input object by creating ASTs from each value
-        // in the PHP object according to the fields in the input type.
-        if ($type instanceof InputObjectType) {
-            return self::convertInputObjectType($value, $type);
-        }
-
```
Comment (crisu83) [inline src/Util/ValueConverter.php]: @Jalle19 Why was this removed?

Comment (Jalle19) [reply src/Util/ValueConverter.php]: Unused, hard to test, didn't work correctly

**Comment by crisu83** [comment]
@Jalle19 what's the status of this PR?
